.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Shall we include this folder in the `.gitignonre` and [remove it from git history](https://fedingo.com/how-to-remove-file-from-git-history/)

You and I use VSCode, but is common to find people in any team that uses different code editors and IDEs.

The questions you might want to ask yourself when committing files that you not so sure are::
- Is this sensitive data?
- Is this particular to my development environment?

Maybe we can extend this list on a later stage, but for now these are the two cases I can think of.